jade/utils/utils.py

```diff
@@ -102,8 +101,8 @@ def _get_module_from_extension(filename, **kwargs):
         mod = json
     elif ext == ".toml":
         mod = toml
-    elif ext in (".yml", ".yaml"):
-        mod = yaml
+    # elif ext in (".yml", ".yaml"):
```

Comment:
We were never using yaml, so we might as well leave this out until someone actually needs it.